### PR TITLE
Add support for Django 2.2.

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

File: django_perf_rec/orm.py [hunk 1]
```diff
@@ -41,18 +46,32 @@ def patch_QuerySet():
...
-                     param_iter = iter(select_params)
-                 else:
-                     param_iter = iter([])
-        -        for name, entry in select.items():
-        +        for name, entry in sorted(select.items()):
-                     entry = force_text(entry)
-                     entry_params = []
-                     pos = entry.find("%s")
-    """)
+if DJANGO_VERSION < StrictVersion("2.2"):
```
Comment (adamchainz) [inline django_perf_rec/orm.py]: The comparison should be for the new version first and you can use `django.VERSION` more simply like

```python
if django.VERSION > (2, 2):
```

File: HISTORY.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ History
 Pending release
 ---------------
 
-.. Insert new release notes below this line
```
Comment (adamchainz) [inline HISTORY.rst]: leave this comment

File: setup.py [hunk 3]
```diff
@@ -10,20 +10,25 @@ def get_version(filename):
     return re.search(r"__version__ = ['\"]([^'\"]+)['\"]", contents).group(1)
 
 
-version = get_version(os.path.join('django_perf_rec', '__init__.py'))
+def get_readme():
```
Comment (adamchainz) [inline setup.py]: Why are you changing all this?

File: .travis.yml [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
+  - 3.6
+  - 3.7
 cache: pip
 
 install:
-  - pip install tox
+  - pip install tox tox-travis
 
-script: tox
+script: tox -r
```
Comment (adamchainz) [inline .travis.yml]: `-r` is not needed on travis since it's a fresh environment each time, `.tox` is not cached

File: requirements.in [hunk 5]
```diff
@@ -4,6 +4,7 @@ flake8-commas
 isort
 multilint
 patchy
+pathlib2
```
Comment (adamchainz) [inline requirements.in]: Why?

Comment (joehybird) [reply requirements.in]: this package is needed for python 3.5. I could add a condition if you want

File: .travis.yml (hunk 4 above)
Comment (joehybird) [reply .travis.yml]: Ok, its just a little habit of mine :) I'll remove it :)

File: setup.py (hunk 3 above)
Comment (joehybird) [reply setup.py]: `with open()...` outside a function is a bit strange to me.

File: HISTORY.rst (hunk 2 above)
Comment (joehybird) [reply HISTORY.rst]: It causes an issue with `multilint` : `warning: check: Explicit markup ends without a blank line; unexpected unindent. (line 226)`

File: django_perf_rec/orm.py (hunk 1 above)
Comment (joehybird) [reply django_perf_rec/orm.py]: done

File: HISTORY.rst (hunk 2 above)
Comment (adamchainz) [reply HISTORY.rst]: You need a blank line after comments in reStructuredText

File: setup.py (hunk 3 above)
Comment (adamchainz) [reply setup.py]: It's not strange. Putting it into a function then directly calling that function has the same effect, but with an extra step to mask it. Please undo all these changes.

File: requirements.in (hunk 5 above)
Comment (adamchainz) [reply requirements.in]: Hmm I'd prefer not to move to a test grid with Python 3.5 etc. at this time. Just focus the PR on Django 2.2 support. I have a template I keep all my OSS projects conforming to, changing one of them away from that creates unnecessary burden on me.

Comment (joehybird) [reply requirements.in]: ok, no pb :) but the readme should be changed to remove the python 3.4-3.5 support, no ?

Comment (adamchainz) [reply requirements.in]: I support Python 3.4+, just don't run the tests on it on Travis. It's a pure Python package, I know the differences from 3.5+ and just don't use them. If there's a compat issue I'll move to grid testing all the Python versions, but I'll do it on all repos at once.

Comment (joehybird) [reply requirements.in]: done :)

File: HISTORY.rst (hunk 2 above)
Comment (joehybird) [reply HISTORY.rst]: It wasn't obvious, thanks for the explanation :)

**Comment by joehybird** [comment]
Hello, I think I've done all your required changes. Is it ok ?

**Comment by adamchainz** [comment]
Just added a tiny commit to re-upgrade pytest, merged, and released as 4.3.0: https://pypi.org/project/django-perf-rec/4.3.0/

Thanks! 🎉

**Pull request merged**